hive/src/main/java/com/netease/arctic/hive/catalog/ArcticHiveCatalog.java

```diff
@@ -137,6 +136,7 @@ protected void doDropTable(TableMeta meta, boolean purge) {
     } catch (TException | InterruptedException e) {
       throw new RuntimeException("Failed to drop table:" + meta.getTableIdentifier(), e);
     }
+    super.doDropTable(meta, purge);
```

Comment:
Why moving droping hive table first?

Reply:
When super.doDropTable is called and purge == true, all directories of the table will be remove. And then when try to drop hive table, the path will not be found